vector/src/main/java/im/vector/app/features/crypto/verification/IncomingVerificationRequestHandler.kt

```diff
@@ -60,9 +62,8 @@ class IncomingVerificationRequestHandler @Inject constructor(
         when (tx.state) {
             is VerificationTxState.OnStarted       -> {
                 // Add a notification for every incoming request
-                val name = session?.getUser(tx.otherUserId)?.displayName
-                        ?: tx.otherUserId
-
+                val user = session?.getUser(tx.otherUserId)
+                val name = user?.displayName ?: tx.otherUserId
```

Comment:
I know this is not new, but `user?.displayName` should be replaced by `user?.getBestName()`